jobs/SDE/Assignments/SDE-FE-Assignment.md

```diff
@@ -0,0 +1,76 @@
+### SDE coding challenge
+
+Here you are! We're happy that you're interested in working for Quantum Inventions! 
+To get a better idea of your current skills, we'd like you to complete a coding challenge - build a web app to display and update resources (e.g - users). 
+The actual requirements are listed further down, but here are the general tech requirements:
+- Make sure your app is runnable on Windows, Mac OS X and Linux
+- Do not include any paid software dependencies
+- Include instructions for setting up and running your application
+- Containerized app is a plus
```

Comment:
Please review this summary section. It has been copied over from the BE task but some of the points dont apply and there might be other points relevant for FE

Reply:
Updated!